windows/binary_security_descriptor.ksy

```diff
@@ -0,0 +1,244 @@
+meta:
+  id: binarysd
```

Comment:
Must match ksy file name stem. It may make sense to have a word `windows` within it: KS has no namespaces.